src/System.CommandLine/Parsing/OptionResult.cs

```diff
@@ -28,13 +28,13 @@ public sealed class OptionResult : SymbolResult
         /// <summary>
         /// The option to which the result applies.
         /// </summary>
-        public Option Option { get; }
+        public CliOption Option { get; }
 
         /// <summary>
         /// Indicates whether the result was created implicitly and not due to the option being specified on the command line.
         /// </summary>
         /// <remarks>Implicit results commonly result from options having a default value.</remarks>
-        public bool IsImplicit => IdentifierToken is null || IdentifierToken.IsImplicit;
+        public bool Implicit => IdentifierToken is null || IdentifierToken.Implicit;
```

Comment:
thank you for renaming all the properties, including even these which were not officially approved yet. It's great for consistency  and should help in the next API review round